src/main/java/edu/kit/hci/soli/controller/LoginController.java

```diff
@@ -28,11 +28,6 @@ public String login(HttpServletRequest request, Model model) {
             model.addAttribute("logout", "You have been logged out");
         }
 
-        CsrfToken csrf = (CsrfToken) request.getAttribute(CsrfToken.class.getName());
-        if (csrf != null) {
-            model.addAttribute("csrf", csrf);
-        }
-
```

Comment:
This was moved to the LoginControllerAdvice since we need it for all forms, not just the login form